fuel-benches/benches/set/blockchain.rs

```diff
@@ -0,0 +1,121 @@
+use super::run_group;
+
+use criterion::Criterion;
+use fuel_core_benches::*;
+use rand::{
+    rngs::StdRng,
+    RngCore,
+    SeedableRng,
+};
+
+pub fn run(c: &mut Criterion) {
+    let rng = &mut StdRng::seed_from_u64(2322u64);
+
+    let mut group = c.benchmark_group("blockchain");
+
+    let cases = vec![
+        1, 10, 100, 1_000, 10_000,
+        100_000,
+        // 1_000_000 - I/O error in BAL; investigate
```

Comment:
Should this be added to the issue tracker for `fuel-core`?

Reply:
I'm investigating that one to understand if its a bug in the benchmark implementation or something more. The early conclusion is for the latter, but I'm not sure yet